src/components/Header_type_1.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Grid, Row, Col } from 'react-flexbox-grid';
+import { routeTo } from 'redux-router-kit';
+import { getTranslator } from '../reducers/translation';
+import { RoundHeaderButton } from './Buttons';
+
+import styles from './Header_type_1.scss';
+import common from '../styles/common.scss';
+
+const HeaderInitialComponent = ({ i18n, targetTo }) =>
+  (<header>
+    <div className={common.widthLimiter}>
+      <div className={styles.header}>
+        <Grid fluid>
+          <Row>
+            <Col lg={1} className={styles.logoContainer}>
+              <div className={styles.logo} />
+            </Col>
+            <Col lg={8} className={styles.mainArea}>
+              <div className={styles.title}>
+                {i18n('Neufund Platform')}
+              </div>
+              <div className={styles.status}>
+                {i18n('Log in')}
+              </div>
+            </Col>
+            <Col lg={3} className={styles.createButton}>
+              <RoundHeaderButton
+                onClick={(event) => {
```

Comment:
Hmm.. Either you use `RegisterLink` and apply a `className` to it to make it look like the round header button, or we keep the current implementation. Both approaches feel about equally messy to me.

Alternatively, we can modify `PredifinedLink` to take a component as a property, then it would look like this:

```
<RegisterLink button={RoundHeaderButton}>{i18n('…')}</RegisterLink>
```